index.html

```diff
@@ -31,11 +33,18 @@ <h1>John Doe</h1>
         </li>
       </ul>
 
-      <img
-        class="hamburger-button"
-        src="./icons/Icon-Menu.png"
-        alt="icon-menu"
-      />
+      <button class="hamburger-button" type="button" onclick="openMenu()"><i class="fas fa-bars fa-2x"></i></button>
+    </nav>
+
+    <nav class="menu-dropdown flex">
+      <div class="flex">
+        <button type="button" class="close-btn" onclick="closeMenu()"><i class="fas fa-times fa-lg"></i></button>
+      </div>
+      <ul>
+        <li onclick="closeMenu()">Portfolio<i class="fas fa-chevron-right fa-xs"></i></li>
+        <li onclick="closeMenu()">About<i class="fas fa-chevron-right fa-xs"></i></li>
+        <li onclick="closeMenu()">Contact<i class="fas fa-chevron-right fa-xs"></i></li>
+      </ul>
     </nav>
```

Comment:
- [ ] Good job implementing the dropdown menu as requested! However, there are a few tweaks required for you to make in this section:
- The menu items should link to actual parts of the page. See [here](https://www.w3schools.com/html/html_links.asp) for information on how to link to items on a page.
- You have some background image repetition and misplacement, kindly fix it.
- The headline also has some issues with the background image decoration. Kindly refer to the Figma design template and make these corrections.

_See images below for reference._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/122838399-e0a33e00-d2aa-11eb-9b05-09915da17f69.png) | ![image](https://user-images.githubusercontent.com/49621279/122838435-f153b400-d2aa-11eb-9f2f-83a684a0eec2.png)
![image](https://user-images.githubusercontent.com/49621279/122838538-19dbae00-d2ab-11eb-813b-00eb12221980.png) | ![image](https://user-images.githubusercontent.com/49621279/122838573-2c55e780-d2ab-11eb-8172-abd000e291ff.png)

Reply:
Thanks!